src/pages/workspace/WorkspaceMembersPage.tsx

```diff
@@ -531,7 +535,7 @@ function WorkspaceMembersPage({personalDetails, route, policy, currentUserPerson
             return null;
         }
         return (
-            <View style={styles.w100}>
+            <View style={[styles.flexRow, styles.gap2, shouldUseNarrowLayout && styles.mb3]}>
```

Comment:
This seems to have caused this: https://github.com/Expensify/App/issues/49749, why did we change the style here? @Guccio163

Reply:
Was it because we needed to acomodate two buttons?

<img width="294" alt="image" src="https://github.com/user-attachments/assets/c6d96660-0f18-4741-9f80-de5ceb2015c1">

Reply:
oh, I see, without this change in desktop the 3-vertical-dots are too far to the right:

<img width="1099" alt="image" src="https://github.com/user-attachments/assets/8e8b0dbc-2f29-480f-a82c-8b7b2b0943ef">